client/state/data-layer/third-party/directly/test/index.js

```diff
@@ -15,7 +13,7 @@ import {
 	DIRECTLY_INITIALIZATION_SUCCESS,
 	DIRECTLY_INITIALIZATION_ERROR,
 } from 'state/action-types';
-import * as analytics from 'state/analytics/actions';
+import * as analytics from 'state/analytics/actions/record';
```

Comment:
Note: this is not a change in the interface of the `state/analytics/actions` module, but rather a quirk with Sinon. It appears it can't handle re-exports the same as regular exports.